packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -175,17 +172,17 @@ mixin SearchControllerMixin<T extends DataSearchStateMixin> {
       matchIndex.value = 1; // first item because [matchIndex] us 1-based
     }
     _activeSearchMatch.value?.isActiveSearchMatch = false;
-    _activeSearchMatch.value = searchMatches.value[activeMatchIndex]
+    _activeSearchMatch.value = searchMatches.value[activeMatchIndex]!
       ..isActiveSearchMatch = true;
     onMatchChanged(activeMatchIndex);
   }
 
   /// Delay to reduce the amount of search queries
   /// Duration.zero (default) disables debounce
-  Duration get debounceDelay => null;
+  Duration? get debounceDelay => null;
```

Comment:
can you move this to the top of the mixin where the other variables and getters are defined?

Reply:
done